hudi-spark/src/test/java/org/apache/hudi/table/NoOpBulkInsertPartitioner.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.table;
+
+import org.apache.hudi.common.model.HoodieRecord;
+import org.apache.hudi.common.model.HoodieRecordPayload;
+import org.apache.spark.api.java.JavaRDD;
+
+public class NoOpBulkInsertPartitioner<T extends HoodieRecordPayload>
```

Comment:
anyway we can use a lambda to implement an anonymuous partitioner instance and avoid this test class

Reply:
I can use Lambda to implement it, however to test it, the class should be able to be loaded(created) by reflection API(```Class.forName(clazzName)```) with its class name. I wasn't sure how classLoader can load lambda class from its class name? 

From my quick test, it doesn't work? do you have any suggestion?

Reply:
You are right.. some cases are tricky. I would just suggest to move this as an inner class for the test itself and leave it be

Reply:
Sure, I will update it.